service/controller/resource/clusterconfigmap/desired.go

```diff
@@ -246,6 +246,9 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) ([]*cor
 		ClusterCIDR:  clusterCIDR,
 		GcpProject:   gcpProject,
 		Provider:     provider,
+		CiliumNetworkPolicy: CiliumNetworkPolicy{
+			Enabled: true,
+		},
```

Comment:
That somehow feels wrong. Which app takes this setting? Can't it be `true` by default in there?

Reply:
The answer is: all the apps that are deployed with this setting, e.g cert-manager